team.py

```diff
@@ -40,4 +42,20 @@ def edit_password(self, input):
             return False
         self.password = password
         ret = "Password successfully changed to " + password
-        return ret
\ No newline at end of file
+        return ret
+
+    def answer_question(self, user_answer):
```

Comment:
user_answer is not a string, but rather a list of strings, so to get the actual answer, you'd probably want to do this:

def answer_question(self, input):
user_answer = input[0]